### ISSUE TITLE
Incorrect sort of exports with as

### ISSUE THREAD

**Issue by sQVe** [lydell/eslint-plugin-simple-import-sort#58]
Hey :wave:,

The new `exports` sort does not handle sorting exports that use the `as` keyword. 
```
export { something, something as default } from './something'
```
will be sorted as:
```
export { something as, defaultsomething } from './something'
```
which obviously is incorrect.

**Comment by lydell** [comment]
Hi! Thanks, good catch! It seems to only happen for `as default` (not `as somethingElse`).

**Comment by lydell** [comment]
Released in v6.0.1.
